CodesmithProjects/TrailBlazers
# Added google map directions and estimated trail completion time to more info page

## Overview

Issue Type


Description

Added a link to google maps with the trail as the destination from the more info page.
Added an estimation on how long a trail will take to bike in minutes on the more info page.

Ticket Item

FEATURE #2: Backend google maps
FEATURE #2: Generating estimation

Steps to Validate Feature

1. Start up the front and back end
2. Go to http://localhost:5173/
3. Input a zip code and get trails
4. Scroll down to click on a specific trail
5. On the new info page, click directions to get a new google maps directions tab
6. Or on the new info page, check the estimated time section
